### ISSUE TITLE
Get an error when choosing a "Main Menu" shortcut

### ISSUE THREAD

**Issue by bykof** [ClipMenu/ClipMenu#1]
Hey I have installed the new version of ClipMenu (v. 1.0.0a1). Everytime I want to choose the Main Menu Shortcut I get this error: 

This shortcut cannot be used because it is already used by the menu item ‘Paste Style’. The key combination ⌥⌘V cannot be used

Hope the information helps you!

**Comment by xjlin0** [comment]
I am happily using ClipMenu-1.0.0a1 in Yosemite and cannot stop smiling -- It' insanely great. :+1: 
Not so sure what is "Main Menu", but if you click the pasteboard icon on the upper right of the entire screen, you can edit/record customized shortcuts under Preference :arrow_right: Shortcuts, including "Main Menu".
The default shortcut ⇧⌘.  works for me (don't forget the dot) to call the main menu.  Just wish there's a way to turn on the "launch at login" option.

**Comment by bykof** [comment]
Yes, you're right. ClipMenu 1.0.0.1a is wonderful. But maybe you doesn't understand my problem. 
I upgraded ClipMenu and had this combination in the old version: ⌥⌘V
In the new version I can't set this combination (⌥⌘V) to call the main menu. Because it says: 

This shortcut cannot be used because it is already used by the menu item ‘Paste Style’. The key combination ⌥⌘V cannot be used

**Comment by xjlin0** [comment]
Did you remove the original preference of old ClipMenu 0.4.3? How about move or delete ~/Library/Application Support/ClipMenu and reboot?

**Comment by bykof** [comment]
I just tried your suggestion but it seems not to work. :( 
Where are the key preferences stored of the old 0.4.3v, maybe I can delete them?
